model_api/python/model_api/adapters/utils.py

```diff
@@ -41,6 +41,8 @@ def from_shape(shape):
             return "CHW" if shape[0] in range(1, 5) else "HWC"
         if len(shape) == 4:
             return "NCHW" if shape[1] in range(1, 5) else "NHWC"
+        if len(shape) == 6:
+            return "?NCTHW" if shape[1] in range(1, 5) else "?NTHWC"
```

Comment:
What does `?` mean?

Reply:
mmaction2 adds one more dimension and it's called `N_crops` in doc string. but I don't know what it is exactly. There is no explanation about that.

Reply:
I dicussed it w/ Jaeguk and we understand what it is. I updated it.